src/main/resources/db/migration/V1_1__InsertData.sql

```diff
@@ -10,7 +10,7 @@ VALUES ('c84936e4-2f0f-11e8-b467-0ed5f89f718b','Community Foundation Of Des Moin
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493824-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Planting Bamboo','We are planting lots of Bamboo',now(), null);
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493950-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Big Hug','We are giving lots of Hugs',now(), null);
 
-insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','apanda@gmail.com','Organization Administrator');
+insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','anthonybrobston@gmail.com','Organization Administrator');
```

Comment:
I don't think this should be committed

Reply:
Once there's more than just me doing development, I'll stop making changes to this file. I know this isn't proper for flyway, however since I'm only running my database in docker on my computer, when I make changes I just blow away the container and recreate it. I can remove my email though too, I'm not sure if that's what you're referring to.